mysql-test/suite/mariabackup/mdev-18438.test

```diff
@@ -0,0 +1,9 @@
+let $basedir=$MYSQLTEST_VARDIR/tmp/mdev-18438;
+mkdir $basedir;
+exec $XTRABACKUP --defaults-file=$MYSQLTEST_VARDIR/my.cnf --backup --extra-lsndir=$basedir/extra_lsndir --stream=xbstream > $basedir/stream.xb 2>/dev/null;
```

Comment:
No redirection to  /dev/null, please. 
If you want to suppress output from `exec`,  do this

    --disable_result_log
     exec $XTRABACKUP ....
    --enable_result_log

like it is used consistently in the whole suite.
This

- will suppress output in case of successful exec

- will  ensure that otuput is dumped in case of error from exec
- will be portable outside of Unix

Reply:
@vaintroub any idea why `extra_lsndir_stream` fails on windows (contributor used `--disable_result_log` there instead of redirecting to dev/null)? Redirecting to dev/null seems to solve the problem.
http://buildbot.askmonty.org/buildbot/builders/winx64-packages/builds/15206/steps/test/logs/stdio

Reply:
No idea why it failed, but if it fails you probably do not want to have binary data in XBSTREAM format on stdout to be intermixed with mariabackup log  on stderr.

As for /dev/null, indeed, there is some magic in mysqltest.c that replaces /dev/null with NUL on Windows.

Anyway, other point is still valid. If you redirect mariabackup's *stderr* to /dev/null or NUL like in mdev-18438.test , and mariabackup fails, you will not find out why from the buildbot log, because its output was unconditionally supressed.